docs/sdk/core/baseapp.md

```diff
@@ -0,0 +1,48 @@
+# BaseApp
```

Comment:
Thanks, I think we can add more later but this is a great start.

Reply:
💯 % agreed and that was my intention. I'll create an issue out of this.